conftest.py

```diff
@@ -36,3 +28,19 @@ def dev_account():
 @pytest.fixture
 def timeout():
     return TIMEOUT
+
+
+def pytest_addoption(parser):
```

Comment:
This still needs to be moved to `tests/marionette` as it's not used by Selenium.